demo_templates/regex.yaml

```diff
@@ -0,0 +1,14 @@
+- key: regex
+  expect:
+    http:
+      method: POST
+      path: /regex
+  actions:
+    - reply_http:
+        status_code: 200
+        body: >
+          {
+            "match first string": "{{.HTTPBody | regexFind "peach"}}",
+            "match and get first group": "{{.HTTPBody | regexFindFirstSubmatch "p([a-z]+)ch" }}",
+            "match and get all groups": "{{.HTTPBody | regexFindAllSubmatch "p([a-z]+)ch" }}"
+          }
```

Comment:
@ziru added examples

Reply:
nit: can we add rendered result as well via comments?  best with the use of `index`.
also, do you have any resource links for golang templates?   mockr users do not need to know golang a ton, but would need to know golang templates.